cocos/platform/winrt/CCWinRTUtils.cpp

```diff
@@ -59,7 +59,6 @@ bool isWindowsPhone()
 #else
     return false;
 #endif
-    return false;
```

Comment:
it is needed or no value will return in ` #if _MSC_VER >= 1900` and `if` sentence returns false